libpages/root.go

```diff
@@ -129,7 +129,7 @@ func (r *Root) MakeFS(
 		if err != nil {
 			return CacheableFS{}, tlf.ID{}, nil, err
 		}
-		tlfFS, err := libfs.NewReadonlyFS(
+		tlfFS, err := libfs.NewFS(
```

Comment:
How does this fix the test?  Doesn't making this readonly improve the performance?

Reply:
I haven't looked into it carefully yet, but I think it's related to the fact that `SubscribeToObsolete` requires it to not be empty, but NewReadonlyFS can get an empty one initially. I was reluctant to simply remove that nonEmpty check for this method, because I was not sure if we need to do something special if the folder is empty. Didn't have a chance to make sure of that yet.

Reply:
Will merge this for now and try to figure this out later!